content/en/docs/instrumentation/net/framework-config.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: ".NET Framework instrumentation configuration"
-linkTitle: ".NET Framework"
+title: .NET Framework instrumentation configuration
+linkTitle: Framework config
```

Comment:
I would say, that it is rare to use  just Framework even if the context is known. It is usually used as a full proper name _.NET Framework_.

Reply:
Yes, this should say `.NET Framework`. It refers to a different runtime/toolset (.NET is a family of runtimes, tools, compilers, etc.)

Reply:
Got it, thanks!

So should I just revert all changes to this file @cartermp?
Or maybe just close the PR?

Reply:
I would say revert the changes to this file and it should be good. If it's easier to just do another PR then that's good too 🙂

Reply:
Ok, so I've reverted the title and linkTitle changes.

As for the website URL, would it make sense to use the following:

- `.../docs/instrumentation/net/framework`

Or do you want to keep the **_stuttered_** `net` that we had before, namely:

- <code>.../docs/instrumentation/<b>net/net</b>framework</code>

Reply:
I'd prefer the latter. Even though it seems kinda weird, `netframework` is unambiguous about what it supports, whereas `framework` can be ambiguous.